grype/matcher/apk/matcher.go

```diff
@@ -135,3 +135,81 @@ func vulnerabilitiesByID(vulns []vulnerability.Vulnerability) map[string][]vulne
 
 	return results
 }
+
+func (m *Matcher) findApkPackage(store vulnerability.Provider, d *distro.Distro, p pkg.Package) ([]match.Match, error) {
+	var matches = make([]match.Match, 0)
```

Comment:
```suggestion
	var matches []match.Match
```

This syntax does _almost_ the same thing, except 1) it's a bit more idiomatic for empty slices, and 2) it helps to avoid unnecessary allocation

Reply:
Also — nit: You might consider moving the declaration of `matches` closer to where it's first used

Reply:
Good to know! Updated both suggestions!